inference_scripts/ep_infer/ep_infer_lm.py

```diff
@@ -67,12 +59,10 @@ def main(args):
     tracker = gs.create_builtin_task_tracker(config, infer.rank)
     infer.setup_task_tracker(tracker)
     device = 'cuda:%d' % infer.dev_id
-    fanout = config.eval_fanout if config.mini_batch_infer else []
-    dataloader = GSgnnEdgeDataLoader(infer_data, infer_data.test_idxs, fanout=fanout,
+    dataloader = GSgnnEdgeDataLoader(infer_data, infer_data.test_idxs, fanout=[],
```

Comment:
For edge prediction, we will not support mini batch?

Reply:
This is for tasks with language model as encoder only. There is no GNN in the model, so that we do not need fanout.